packages/third-party/tailwind-preset/src/lynx.ts

```diff
@@ -2,13 +2,14 @@
 // Licensed under the Apache License Version 2.0 that can be found in the
 // LICENSE file in the root directory of this source tree.
 
-import { display, position, underline } from './plugins/lynx/index.js';
+import { display, position, textDecoration } from './plugins/lynx/index.js';
 
 /**
  * Should be used with Tailwind JIT and configured with purge, otherwise the bundle will be too large.
  */
 export default {
-  plugins: [position, underline, display],
+  content: ['./**/*.{js,ts,jsx,tsx,css}'],
```

Comment:
Why do we need to add `*.css` to content?

Reply:
@upupming up 
![image](https://github.com/user-attachments/assets/8a52b61b-2bb0-4b46-a3e4-1e3f0550c037)

Reply:
> Why do we need to add `*.css` to content?

packages/third-party/tailwind-preset/src/__tests__/styles.css